### PR TITLE
ficks sum, grammers

### PR DESCRIPTION
**Pull request by robbkidd**
Some grammar corrections for the docs lie herein.

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: www/source/docs/contribute-help-build.html.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ The remainder of this document describes some of the areas of the product that c
 
 ## More Topologies
 
-Habitat ships on day one with [two topologies](/docs/run-packages-topologies): standalone, and leader-follower. There are undoubtedly more that are needed in order to represent the semantics of real systems.
```
Comment (mwrock) [inline www/source/docs/contribute-help-build.html.md]: I certainly dont think this is blocking but technically I don't think this comma is necessary/correct. You would want it if the clause after the and could be a complete sentance or if it was a list of atleast 3 (this is controversial). That all said I'm totally ready to be told I'm wrong and honestly I don't care passionately about this :)

**Comment by mwrock** [comment]
@thesentinels approve

**Comment by thesentinels** [comment]
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

**Comment by thesentinels** [comment]
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/184588225).

**Comment by thesentinels** [comment]
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/184588225).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

gif-keyboard-3280869874741411265

File: www/source/docs/contribute-help-build.html.md (hunk 1 above)
Comment (robbkidd) [inline www/source/docs/contribute-help-build.html.md]: ... I think we are in agreement. With only two items left in the list, that's why I removed it. Or am I misunderstanding your comment?

**Comment by fnichol** [comment]
I couldn't even read the commit message title. Had to say it out loud, haha.

**Pull request merged**